GOVERNANCE.md

```diff
@@ -165,6 +174,84 @@ We have a reasonably liberal approach to code review and merging. We value quick
 - All code must go through Pull Requests (PR) and pass status checks before being merged. If a PR is merged that breaks `main` due to the branch not being up-to-date, then it should either be reverted or a quick fix merged as a separate PR.
 - If a PR is against code you have previously committed and is either small changes, bug fixes, or refactors, then you're free to merge it without any review. However, if you need more confidence in your changes, you can wait for approval from another maintainer or core contributor.
 
+## Financial Contributions
+
+Biome is in the fortunate position that people care enough about it that they want to invest financially in its future. However, with money comes responsibility, so we are committed to the following.
+
+There are two ways to fund Biome and its development: sponsorship and funded bounties, although we strongly prefer sponsorship.
+
+### Sponsorship
+
+Sponsorship is the first and foremost way of making financial contributions to Biome. Sponsorship can be
+done through a one-time donation, or through recurring donations. We offer incentives for certain levels of recurring donations.
+
+To learn more about sponsorship incentives, or to make a donation, please visit our [Open Collective page](https://opencollective.com/biome).
+You can also sponsor the project through [GitHub](https://github.com/sponsors/biomejs).
+
+### Community-Funded Bounties
+
+Bounties are the second approach towards funding Biome development. Unlike a sponsorship, bounties are directed towards the completion of a specific task. Tasks include, but are not limited to, the development of a feature or the fixing of a bug.
```

Comment:
So if someone wants to adopt Biome but finds there's an issue blocking them, I think it's very natural to want to fund said issue. I'm not really sure I understand the objection? Of course we may wish for them to fix bugs themselves, but we have to recognize not everyone has the skills or time to do so. If anything, I'd say financial contributors are part of the community too :)

There's also another reason why I'm hesitant towards making a distinction that would disallow bounties on bug fixes: Developers and users regularly argue about what should be considered a bug and what is a (missing) feature. By saying one can be funded and the other cannot, we even create a financial incentive for these arguments to take place. I think it's not a good idea to make such a distinction.

Reply:
This could incentivize behaviours where  people would fix bugs only if paid, which is something I am really afraid of and I wouldn't want to see.

If someone wants a bug fixed ASAP, I would rather see them sponsor the project so we could afford someone working part/full time on maintenance or bug fixes.

I've seen people implementing features (in Biome) and then disappearing, and **I** had to fix the bugs.

It's not about the users that want their bugs fixed, but it's about having the right people and resources to fix a bug.

Reply:
Doc is updated :)